### ISSUE TITLE
[ Feature req ] Theme following and JSONPath search

### ISSUE THREAD

**Issue by 7system7** [fleury08/prettifier#36]
Firstly, thank you for this app. I use it under flatpak.

It would be good, if 
- the UI can follow the system theme 
- and day/night settings (I use GNOME and arch btw. :smile: )
- another good feature could be, if we can search in the JSON w/ JSONPath. (Just like in i.e. Insomnia)

![Screenshot from 2022-07-12 13-48-10](https://user-images.githubusercontent.com/3964568/178483072-10aff0c6-121b-41e7-8fe7-f937a8828993.png)

**Comment by fleury08** [comment]
Hi, the app should be following the system theme, at least what I have tested on my system and VBox.
Now It is set to follow the system theme, but I think that would be better to give a choice [Auto, Dark, Light].
But haven't tested on vanilla Gnome, at least not thoroughly, god forbid running on Arch :smile: 
You can check by yourself If the app isn't forced to use the light theme with the app Darkbar or similar.

I will test It in VBox on some Arch+Gnome distro and figure It out, also I will add the manual option.
JSONPath would be a nice addition.

Thanks for the feedback.

**Comment by 7system7** [comment]
Hi, I use GNOME 42.x but cause' the flatpak it does not matter too much. :) I made some settings on flatpak globally to follow the system theme. (adwaita) I have a Tweaks setting for legacy applications the others use the vanilla adwaita. Sadly the Prettifier is an exception. (Maybe because the Elementary roots, I do not know, I do not use other Elementary app except this.)

![Screenshot from 2022-07-12 14-19-05](https://user-images.githubusercontent.com/3964568/178488463-de56783c-10ad-4433-9ad9-0395a4bdcded.png)

**Comment by fleury08** [comment]
Great, thank you for the info. The system theme following outside of elementaryOS should not be a problem. I guess you are compiling the app and don't use the elementary flatpak repo?

**Comment by 7system7** [comment]
Nope, I do not compile it. 🙂 This is one of the few apps what I use from the Software center - and from the flatpak repo.

![Screenshot from 2022-07-12 14-30-06](https://user-images.githubusercontent.com/3964568/178490309-e3ff5e1b-c779-467a-a495-55b5a855dc69.png)
